examples/tls-logger/server.cpp

```diff
@@ -156,10 +169,15 @@ class server
     void start()
     {
         listen(_srv,3);
-        _thread = std::thread([this]
-        {
+	// iterate for num threads
+	for (std::vector<std::thread>::iterator it = thread_list.begin();
```

Comment:
If you take the suggestion I made in the constructor, you can do something like:

```c++
while(thread_list.size() < thread_list.capacity())
{
    thread_list.push_back(std::thread(...));
}
```
This is a faster solution since you will only be constructing each thread once.